.devcontainer/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM pangeo/base-image:2023.06.20
```

Comment:
Cool. For later: Is there a way to automatically bump this?

Reply:
Possibly, maybe do an automated check and make a PR when there's a new version?

Reply:
I'd be surprised there's no existing Docker image bump workflow?

Reply:
Not sure... need further investigation on that.

Reply:
Forgot about dependabot! Okay, I've added the config on a `dependabot.yml`.